lib/db.js

```diff
@@ -37,6 +42,20 @@ function createObservationsReplicationStream () {
   return observationsDb.log.replicate()
 }
 
+function importBulkOsm (xmlReadableStream, cb) {
+  if (osmOrgDb) { osmOrgDb.db.close() }
```

Comment:
`close()` is an async method -- you might need to pass it a callback and wait for it to return before doing the rimraffing below.